### ISSUE TITLE
Duplicated Scheme

### ISSUE THREAD

**Issue by elerbe** [Burgyn/MMLib.SwaggerForOcelot#117]
**Describe the bug**
Schemas in generated request url are duplicated.

This bug is same as #53. Which was fixed in [commit](https://github.com/Burgyn/MMLib.SwaggerForOcelot/pull/67/commits/b535b67365aae02c825efcc263a42a2ec41bc61c)
But in version 2.0.2 was changed again: [commit](https://github.com/Burgyn/MMLib.SwaggerForOcelot/commit/ac38873b326d6b84e59462140a55db2882351043)

**To Reproduce**
My stack: 
* .net core 3.1 
* Swagger 5.x
* Swagger configuration : 
```
.UseSwagger(c =>
{ 
    c.RouteTemplate = routePrefix + "/{documentName}/swagger.json"; 
    c.SerializeAsV2 = true;
});
```

**Comment by Burgyn** [comment]
Hi,

how is possible that you have scheme (`http`) in Host header?

Can you provide more information, which can help simulate this behaviour?

Thanks.

**Comment by natiox** [comment]
Same issue here.
Under generated page "Try it out"
Request URL: https://https://localhost:44325/api/user

**Comment by Burgyn** [comment]
Hi @natiox

please can you provide demo? Because I cannot reproduce it.

Thanks.

**Comment by natiox** [comment]
[SwagForOcelotProblemDemo.zip](https://github.com/Burgyn/MMLib.SwaggerForOcelot/files/4983063/SwagForOcelotProblemDemo.zip)
Attached is a demo solution.  Set both projects to start.  Requires Azure Emulator to be installed.  This demo was failing to run the first time.  I stopped it and tried again and it ran.

**Comment by Burgyn** [comment]
Hi @natiox 

I would like to help, but I have trouble reproducing it. If I use your example, everything works.
I believe there is a problem. But I would like to know the conditions when it occurs.

If you have an idea of ​​what may be different then write.

![image](https://user-images.githubusercontent.com/5930822/88660455-080f7c00-d0d7-11ea-9060-b329abf1d45c.png)

**Comment by natiox** [comment]
In that demo project, try Swagger on the gateway instead.  https://localhost:44325/swagger/

**Comment by Burgyn** [comment]
Hi @natiox,
Thanks. My mistake 😳

I'm not sure why this happens with your configuration. 
But as a workaround can try to set the `HostOverride` property to `""`.

```json
  "SwaggerEndPoints": [
    {
      "Key": "user",
      "HostOverride": "",
      "Config": [
        {
          "Name": "User API",
          "Version": "v1",
          "Url": "http://localhost:7071/api/swagger.json"
        }
      ]
    }
  ],
```

**Comment by natiox** [comment]
Did some debugging with the project source.  Looks like you are emitting Swagger 2.0.  According to the [Swagger 2.0 specs](https://swagger.io/docs/specification/2-0/api-host-and-base-path/), 

> _host_ is the domain name or IP address (IPv4) of the host that serves the API. It may include the port number if different from the scheme’s default port (80 for HTTP and 443 for HTTPS). **Note that this must be the host only, without http(s):// or sub-paths.**


Possible fix:
SwaggerJsonTransformer.cs
```c#
        private static void AddHost(JObject swagger, string swaggerHost)
        {
            var hostUri = new Uri(swaggerHost);
            swagger.Add(SwaggerProperties.Host, hostUri.Authority);
        }
```

**Comment by Burgyn** [comment]
Hi,

this bug is fix in version `2.5.1`. Great thanks to @jacojvv-dev and @natiox
